### PR TITLE
R4R: Basic log

### PR DESCRIPTION
**Pull request by chengwenxi**

### CODE REVIEW

File: modules/stake/handler.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/common"
 	tmtypes "github.com/tendermint/tendermint/types"
+	"fmt"
```
Comment (HaoyangLiu) [inline modules/stake/handler.go]: go imports file

File: app/baseapp.go [hunk 2]
```diff
@@ -599,10 +599,6 @@ func (app *BaseApp) getContextForTx(mode RunTxMode, txBytes []byte) (ctx sdk.Con
 
 // Iterates through msgs and executes them
 func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, mode RunTxMode) (result sdk.Result) {
-	if app.runMsg != nil {
-		return app.runMsg(ctx, msgs, mode)
-	}
-
```
Comment (wukongcheng) [inline app/baseapp.go]: Should also remove:
`func (app *BaseApp) SetRunMsg(runMsg RunMsg)` and `runMsg           RunMsg`.

File: modules/stake/handler.go [hunk 3]
```diff
@@ -34,6 +35,8 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 
 // Called every block, update validator set
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) (validatorUpdates []abci.ValidatorUpdate) {
+	logger := ctx.Logger().With("handler", "endBlock").With("module", fmt.Sprintf("iris/%s", "stake"))
+	logger.Info("stake end block")
```
Comment (wukongcheng) [inline modules/stake/handler.go]: logger.Debug("stake end block")

File: modules/stake/handler.go [hunk 4]
```diff
@@ -34,6 +34,8 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 
 // Called every block, update validator set
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) (validatorUpdates []abci.ValidatorUpdate) {
+	logger := ctx.Logger().With("handler", "endBlock").With("module", "iris/stake")
```
Comment (HaoyangLiu) [inline modules/stake/handler.go]: ctx = ctx.WithLogger(ctx.Logger().With("handler", "endBlock").With("module", "iris/stake"))

File: modules/mint/abci_app.go [hunk 5]
```diff
@@ -9,7 +9,7 @@ import (
 
 // Called every block, process inflation on the first block of every hour
 func BeginBlocker(ctx sdk.Context, k Keeper) sdk.Tags {
-	logger := ctx.Logger().With("module", "x/mint")
+	logger := ctx.Logger().With("handler", "beginBlock").With("module", "iris/mint")
```
Comment (HaoyangLiu) [inline modules/mint/abci_app.go]: ctx = ctx.WithLogger(ctx.Logger().With("handler", "beginBlock").With("module", "iris/mint"))

**Pull request merged**